src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -821,6 +821,21 @@
      */
     String getMSIClientId();
 
+    /**
+     * Sets the Azure Active Directory Application Client ID.
```

Comment:
From ODBC docs:

When KeyStoreAuthentication = KeyVaultPassword, set this value to a valid Azure Active Directory User Principal Name.
When KeyStoreAuthetication = KeyVaultClientSecret set this value to a valid Azure Active Directory Application Client ID

Reply:
Please set Javadoc to this

```
    /**
     * Sets the value for the connection property 'keyStorePrincipalId'.
     * 
     * @param keyStorePrincipalId
     * 
     *        <pre>
     *        When keyStoreAuthentication = keyVaultPassword, set this value to a valid Azure Active Directory User Principal Name.
     *        When keyStoreAuthentication = keyVaultClientSecret, set this value to a valid Azure Active Directory Application Client ID.
     *        When keyStoreAuthentication = keyVaultManagedIdentity, set this value to a valid Azure Active Directory Application Object ID (optional, for user-assigned only).
     *        </pre>
     */
```